src/test/java/org/jitsi/meet/test/pageobjects/web/SettingsDialog.java

```diff
@@ -9,25 +9,31 @@
 public class SettingsDialog
 {
     /**
-     * CSS selectorsto be used for finding WebElements within the
+     * CSS selectors to be used for finding WebElements within the
      * {@link SettingsDialog}.
      */
     private final static String CANCEL_BUTTON = "#modal-dialog-cancel-button";
     private final static String DISPLAY_NAME_FIELD
         = "#setDisplayName";
     private final static String EMAIL_FIELD = "#setEmail";
     private final static String FOLLOW_ME_CHECKBOX = "[name='follow-me'] ~ div";
-    private final static String MORE_TAB
-        =  "//div[contains(@class, 'settings-dialog')]//li[text()='More']";
     private final static String OK_BUTTON = "#modal-dialog-ok-button";
-    private final static String PROFILE_TAB
-        =  "//div[contains(@class, 'settings-dialog')]//li[text()='Profile']";
     private final static String SETTINGS_DIALOG = ".settings-dialog";
     private final static String START_AUDIO_MUTED_CHECKBOX
         = "[name='start-audio-muted'] ~ div";
     private final static String START_VIDEO_MUTED_CHECKBOX
         = "[name='start-video-muted'] ~ div";
-    
+
+    /**
+     * XPath selectors to be used for finding tab label elements within the
+     * {@link SettingsDialog}. XPath is used so tabs can be found based on tab
+     * text, which CSS selectors cannot do.
+     */
+    private final static String MORE_TAB
```

Comment:
as you can see above those constants used to be sorted alphabetically, so please lets stick to that

Reply:
I split them on purpose to separate CSS selectors from Xpath selectors. Should they still be merged into one group?

Reply:
I'd say yes, because the IDE would not be able to figure this out. If you want to keep them together maybe add a prefix